microsoft/botframework-solutions
# [BotBuilder Libs][TypeScript] Implement test main structure

## Description
The following changes were implemented in `botbuilder-skills` and `botbuilder-solutions`
- Implement `mocha`
- Implement `nyc` for **code coverage**
- Add `test report`
- Create test main structure

## Testing Steps

1. Go to `botframework-solutions\lib\typescript\` and open `botbuilder-skills` or `botbuilder-solutions`, it's indistinct
1. Run `npm install`
1. Run `npm run build`
1. Run `npm run test` and check that mocha has attempted to run the test
1. Run `npm run coverage` and check the output for code coverage

## Checklist
